### ISSUE TITLE
usb crash

### ISSUE THREAD

**Issue by riflexpuma** [Lovely-XPP/Dell-Latitude-E7480-Hackintosh#23]
usb install crash after 30 second and pc restart. I tried "If crashed in installing macOS from USB, try Enable UEFI->Quirks->ReleaseUsbOwnership, thanks @krzysinek (#22)" but i have no result. have you same idea? thank you and sorry for my english

**Comment by riflexpuma** [comment]
Depend from different CPU? My hardware is similar, but the cpu is i5 6330u.
thanks.

**Comment by Lovely-XPP** [comment]
Possible (mainly depends on the GPU settings), May you provide more detail information, such as screen shot.

**Comment by riflexpuma** [comment]
excuse me, I wrong. My processor is i5 7300U, graphic card hd 620. I try to boot in verbose mode and make a foto. thanks

**Comment by Lovely-XPP** [comment]
It seems your hardware is same as me, it may not be the problem of GPU settings. How about BIOS Settings, are parameters set as same as me?

**Comment by riflexpuma** [comment]
yes, same. I don't know what I do
yes the hardware is the same....... only processor change.My is i5

**Comment by Lovely-XPP** [comment]
If you have difficulty in English, you can use Chinese : ) 
有可能是别的驱动问题，在出现这个你截图的这个问题前还有别的提示嘛，这个提示信息其实没啥用

**Comment by riflexpuma** [comment]
I understood everything! :)
regarding the kexts in the config.plist with the kexts for the intel wifi, do i have to remove the broadcom kexts? or can I leave everything?

**Comment by riflexpuma** [comment]
![WhatsApp Image 2023-03-14 at 17 19 19](https://user-images.githubusercontent.com/127793258/225070938-8bb0078d-c11d-4c82-9650-ba15fbc4f52a.jpeg)

**Comment by Lovely-XPP** [comment]
> I understood everything! :)
> 
> regarding the kexts in the config.plist with the kexts for the intel wifi, do i have to remove the broadcom kexts? or can I leave everything?

No, if you are using Intel Wi-Fi, you just use the config for Intelwifi without any changes, I did everything done.

**Comment by riflexpuma** [comment]
![Immagine WhatsApp 2023-03-14 ore 15 08 49](https://user-images.githubusercontent.com/127793258/225075626-6de731cf-7a76-4154-a2bd-b4735a1c4ccb.jpg)

**Comment by riflexpuma** [comment]
![Immagine WhatsApp 2023-03-14 ore 17 37 37](https://user-images.githubusercontent.com/127793258/225075889-c9f6cab4-1150-4db2-880a-47566d6e8da0.jpg)

**Comment by riflexpuma** [comment]
have you idea?

**Comment by Lovely-XPP** [comment]
I think you can try to disable USBMap.kext. Maybe it is the problem of USB port. By the way, you may check your MacOS installation.

**Comment by riflexpuma** [comment]
I formatted and tried with various images. ventura and monterey. always panic. i try to disable usbmap?

**Comment by Lovely-XPP** [comment]
Yep, have a try. By the way, can you provide all hardware information including hard disk, voice card etc.

**Comment by riflexpuma** [comment]
kernel panic after disable usbmap.kext

**Comment by riflexpuma** [comment]
![Immagine 2023-03-14 181542](https://user-images.githubusercontent.com/127793258/225086078-aacc6eb1-aadf-406b-bd79-7a318075ae96.jpg)
![Immagine 2023-03-14 181641](https://user-images.githubusercontent.com/127793258/225086092-a2689ea5-a085-450c-a452-c03fb1692ede.jpg)

**Comment by Lovely-XPP** [comment]
Okey, it's very night here, I will check it on the morning, and give some possible solution.

**Comment by riflexpuma** [comment]
![Immagine WhatsApp 2023-03-14 ore 18 47 55](https://user-images.githubusercontent.com/127793258/225093642-3fd56658-db41-4317-be98-81497a61c4c8.jpg)
with your config.plist for intel wifi, without having touched anything, now the pc has not restarted. stuck with this screen. all dead. I had to shut down from the button on the pc

**Comment by Lovely-XPP** [comment]
I think the main problem is 
```
Lilu patcher: @ falled to solve __ZN31AppleIntelFramebufferController17IsTypeCOnlySystemEv, err 2
WhateverGreen Igfx: @ ICOD: Failed to route the function IsTypeCOnlySystem
```
I have some ideas:
- (Most Possible as I think) Maybe you can try OC 0.8.9 and lower.
- Check GPU config setting and I will search some information about that.

**Comment by Lovely-XPP** [comment]
Here is my suggestion: 
Enable Booter -> Quirks -> EnableWriteUnprotector & disable MIMO

**Comment by riflexpuma** [comment]
ok, I'll try as soon as possible.  thanks

**Comment by riflexpuma** [comment]
it occurred to me that my monitor is uhd touch screen and not fullhd.  should i change some settings?

**Comment by Lovely-XPP** [comment]
No, you needn’t. : )

**Comment by Lovely-XPP** [comment]
Then try :
1. PlatformInfo -> Genernic -> ProcessType set to 0.
2. ReEnable MIMO

**Comment by riflexpuma** [comment]
finally I found this :[
](https://osxlatitude.com/forums/topic/17444-latitude-7480-kaby-lake-opencore-and-clover-packs-for-bigsur-monterey-and-ventura-beta/)

The efi work for my hardware, work all, except touchscreen.... can you help me?

**Comment by riflexpuma** [comment]
[[]](https://osxlatitude.com/forums/topic/17444-latitude-7480-kaby-lake-opencore-and-clover-packs-for-bigsur-monterey-and-ventura-beta/)(url)
Finally I found this efi. Work for my hardware. work all except touchscreen. can you help me?

**Comment by Lovely-XPP** [comment]
No problem, Could you upload the EFI files, because I can not download the EFI on the website.

**Comment by riflexpuma** [comment]
[[]](https://drive.google.com/file/d/1iKQ40PlBV-kVyHVCATGkO2AHL8RIURie/view?usp=share_link)(url)

thank you.if it can be useful, with this efi the thunderbolt also works with the computer on. I look forward to your touch screen tips  thanks!!!

**Comment by Lovely-XPP** [comment]
By the way, I would like to know if everything works well by using the EFI you provided, including sleeping and wake etc?
I want to update the repo with the ACPI Setting in the EFI you provided.

**Comment by Lovely-XPP** [comment]
I have modified the EFI you provided, touch screen should be ready for use.

https://1drv.ms/u/s!Arj2Y3XYg4Imkzj2L9B2i1AWdHP5?e=aQO9ZG

**Comment by riflexpuma** [comment]
By the way, I would like to know if everything works well by using the EFI you provided, including sleep, wake etc?
I want to update the repo with the ACPI Setting in the EFI you provided. 
Yes works well

I have modified the EFI you provided, touch screen should be ready for use.

isn't. is your originl efi. no mine.  kerne panic

**Comment by Lovely-XPP** [comment]
Do you use the config `config-Intel-wirelss-card.plist` ?

**Comment by riflexpuma** [comment]
![Uploading D5D92E28-BDBC-48E0-83B0-EB200E7E36AE.jpeg…]()

**Comment by Lovely-XPP** [comment]
I know the problem, it is something about Intel Bluetooth. For macOS Monterey and newer, IntelBTPatcher should be added to kexts kernel.

**Comment by Lovely-XPP** [comment]
I have modified the config file, and ACPI folders is the same as your EFI.

**Comment by Lovely-XPP** [comment]
I updated the repo with your EFI just now, including changing ACPI folders, adding IntelBTPatcher.kext and update config.plist. If it is convenient, could you help me to test if everything works well?
Thank you very much.

**Comment by riflexpuma** [comment]
propertree said that some kext are double

**Comment by Lovely-XPP** [comment]
Yes, some kexts will duplicate because different version of MacOS should have different version kexts. Therefore, it is normal to have duplicate kexts, but these kexts have different name.

**Comment by riflexpuma** [comment]
system don't start. nothing to do. with my efi i'm writing here from hackintosh, but the touch screen is dead

**Comment by Lovely-XPP** [comment]
> system don't start. nothing to do.

With this Repo EFI, system can not start with kernel panic or just can not start?

**Comment by riflexpuma** [comment]
kernel panic after 30 secons

**Comment by riflexpuma** [comment]
nothing to do. panic. mac os start only with my efi

**Comment by riflexpuma** [comment]
without touchscreen

**Comment by Lovely-XPP** [comment]
okey, I try to only modify your EFI config. Later I will upload.

**Comment by Lovely-XPP** [comment]
Maybe it is the problem of USB port, you can try to DIY your usb port by using USBMap:

https://github.com/corpnewt/USBMap
